util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -437,12 +437,20 @@ macro (seqan_configure_cpack_app APP_NAME APP_DIR)
   # The following include automates the MS Redistributable installer.
   include (InstallRequiredSystemLibraries)
 
+  SET(CPACK_GENERATOR "ZIP")
```

Comment:
According to the package site, we provide zip files for linux and bsd as well.
http://packages.seqan.de/razers3/
http://packages.seqan.de/yara/

Would you like to change this?
I will change it if you confirm this.

Thanks~!